sdk/core/core-amqp/src/errors.ts

```diff
@@ -519,8 +537,10 @@ export function isSystemError(err: any): boolean {
   if (
     err.code &&
```

Comment:
Can we factor this massive conditional out of the if's test expression?

Reply:
I can remove the `if` statement and just return the result instead? 😄 

I'll make this more readable.